Lib/concurrent/futures/process.py

```diff
@@ -492,6 +492,9 @@ def terminate_broken(self, cause):
         for p in self.processes.values():
             p.terminate()
 
+        # Prevent queue writing to a pipe which is no longer read.
```

Comment:
What would be writing to the pipe in this context?

Reply:
Internally, `multiprocessing.queues.Queue` uses a thread to write data from an internal deque into the pipe. It gets stuck [here](https://github.com/python/cpython/blob/da4912885f11f525a82a83f795ebffba06560e13/Lib/multiprocessing/queues.py#L250). 

https://github.com/python/cpython/issues/94777 has a good write up of exactly what's happening.

Reply:
Thanks for the explanation. Can a reference to the GH issue be added here?

Reply:
Done.